Test failing after netMHCpan update

Using netMHCpan 3.0 as the netMHCpan binary causes this failure:

```
FAIL: test.test_mutant_epitope_predictions_class1.test_epitope_prediction_without_padding
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/julia/Envs/topiary/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/julia/code/topiary/test/test_mutant_epitope_predictions_class1.py", line 68, in test_epitope_prediction_without_padding
    eq_(len(strong_binders), 4)
AssertionError: 5 != 4
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -48,6 +48,7 @@ install:
   - pip install coveralls
   - pyensembl install --release 75 --species human
   - pyensembl install --release 84 --species human
+  - pyensembl install --release 85 --species human
 
 script:
   - nosetests test --with-coverage --cover-package=topiary && ./lint.sh

--- a/test/test_mutant_epitope_predictions_class1.py
+++ b/test/test_mutant_epitope_predictions_class1.py
@@ -65,7 +65,7 @@ def test_epitope_prediction_without_padding():
         for epitope_prediction in output_without_padding
         if epitope_prediction.value <= 500.0
     ]
-    eq_(len(strong_binders), 4)
+    eq_(len(strong_binders), 5)
 
 
 @raises(ValueError)

